docs/changelog/75981.yaml

```diff
@@ -0,0 +1,9 @@
+pr: 75981
+summary: Bump bundled JDK to 16.0.2
+area: Packaging
+type: upgrade
+issues: []
+versions:
```

Comment:
@pugnascotia What's the purpose of enumerating versions in the YAML file? When we generate the release notes, we're doing so for a _particular_ version, right? They'll show up in other notes simply by means of backporting this file to that branch so this data seems a bit superfluous, no?

Reply:
At a minimum you need the patch release, since e.g. the `7.15` branch will contain changelogs for all the `7.15.*` releases, and you need to be able to say which release specifically included the change. I suppose including all the versions is a bit redundant - I was aiming the make the file "complete", in that you'd be able easily tell if the change was a backport or not. But I suppose you could always hunt through GitHub for that information.

There's also the situation where a change is backported all the way to e.g. `6.8`. The implementation in `GenerateReleaseNotesTask` in ES would only include a changelog file in notes for the `6.8` release.

So we could change HOMER to pick just the earliest version label. WDYT?